### PR TITLE
Add workaround for kernel bug

### PR DESCRIPTION
**Pull request by Jauler**
### Problem
We have been observing `iptables` process being killed by kernel in the testing environment. Recently  we have added [kernel logs capturing](https://github.com/NordSecurity/libtelio/pull/725), and observed that we are hitting NULL pointer deref. With the following stack trace:
```
[Tue Aug  6 01:19:33 2024 <    0.000194>]  ? __die_body.cold+0x1a/0x1f
[Tue Aug  6 01:19:33 2024 <    0.000201>]  ? page_fault_oops+0xd2/0x2b0
[Tue Aug  6 01:19:33 2024 <    0.000197>]  ? exc_page_fault+0x70/0x170
[Tue Aug  6 01:19:33 2024 <    0.000200>]  ? asm_exc_page_fault+0x22/0x30
[Tue Aug  6 01:19:33 2024 <    0.000198>]  ? iptable_nat_table_init+0xe1/0x168 [iptable_nat]
[Tue Aug  6 01:19:33 2024 <    0.000202>]  xt_find_table_lock+0x127/0x1b0 [x_tables]
[Tue Aug  6 01:19:33 2024 <    0.000204>]  xt_request_find_table_lock+0x1a/0x70 [x_tables]
[Tue Aug  6 01:19:33 2024 <    0.000202>]  get_info+0xbe/0x390 [ip_tables]
[Tue Aug  6 01:19:33 2024 <    0.000199>]  ? apparmor_capable+0x74/0x170
[Tue Aug  6 01:19:33 2024 <    0.000200>]  do_ipt_get_ctl+0x74/0x3b0 [ip_tables]
[Tue Aug  6 01:19:33 2024 <    0.000200>]  nf_getsockopt+0x44/0x80
[Tue Aug  6 01:19:33 2024 <    0.000206>]  ip_getsockopt+0x80/0xc0
[Tue Aug  6 01:19:33 2024 <    0.000198>]  __sys_getsockopt+0xc7/0x1e0
[Tue Aug  6 01:19:33 2024 <    0.000197>]  __x64_sys_getsockopt+0x1b/0x30
[Tue Aug  6 01:19:33 2024 <    0.000196>]  do_syscall_64+0x55/0xb0
[Tue Aug  6 01:19:33 2024 <    0.000193>]  ? do_fcntl+0x437/0x690
[Tue Aug  6 01:19:33 2024 <    0.000195>]  ? audit_filter_inodes.part.0+0x2e/0x120
[Tue Aug  6 01:19:33 2024 <    0.000204>]  ? audit_reset_context+0x232/0x300
[Tue Aug  6 01:19:33 2024 <    0.000197>]  ? exit_to_user_mode_prepare+0x44/0x1f0
[Tue Aug  6 01:19:33 2024 <    0.000194>]  ? syscall_exit_to_user_mode+0x1e/0x40
[Tue Aug  6 01:19:33 2024 <    0.000193>]  ? do_syscall_64+0x61/0xb0
```
Which looks almost identical to the one reported here: https://lore.kernel.org/netdev/20240731213046.6194-3-pablo@netfilter.org/T/#m597226ed32420a20bca5be58cf1c21073f06083b

This PR introduces a workaround for the issue by delaying iptables initialization for several seconds at the bootup. 

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Comment by matislovas** [comment]
Will the `docker-compose` script strictly exit _after_ the script? Because if it won't, I guess we need to add some another pre-condition check :smile:

**Comment by Jauler** [comment]
> Will the `docker-compose` script strictly exit _after_ the script? Because if it won't, I guess we need to add some another pre-condition check 😄

Ui, this is a good point :D I forgot about this :D. entrypoint is left for execution forever, and AFAIK there is no wait. Let me try to fix that.

**Comment by matislovas** [comment]
+1

File: nat-lab/docker-compose.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ services:
     volumes:
       - ./:/sync
       - ../:/libtelio
+    healthcheck:
+      test: "ls /ready"
```
Comment (stalowyjez) [inline nat-lab/docker-compose.yml:40]: Shouldn't this healthcheck be added to all of the containers?

File: nat-lab/docker-compose.yml [hunk 2]
```diff
@@ -414,6 +418,8 @@ services:
     volumes:
       - ./data/nordderper/config1.yml:/etc/nordderper/config.yml
     working_dir: /etc/nordderper
+    healthcheck:
+      test: "curl http://localhost:8765"
```
Comment (stalowyjez) [inline nat-lab/docker-compose.yml:422]: Hmm, maybe some comment why 8765? (Are these just descending digits? ^^' If so, maybe 1234 is more obvious)

Comment (Jauler) [reply nat-lab/docker-compose.yml:422]: 8765 is configured derp port https://github.com/NordSecurity/libtelio/blob/main/nat-lab/data/nordderper/config1.yml#L3

File: nat-lab/docker-compose.yml (hunk 1 above)
Comment (Jauler) [reply nat-lab/docker-compose.yml:40]: good point!

File: nat-lab/docker-compose.yml (hunk 2 above)
Comment (Jauler) [reply nat-lab/docker-compose.yml:422]: I will add comment which points to `nat-lab/data/nordderper/config*.yml`

File: nat-lab/docker-compose.yml (hunk 1 above)
Comment (Jauler) [reply nat-lab/docker-compose.yml:40]: Fixed

File: nat-lab/docker-compose.yml (hunk 2 above)
Comment (Jauler) [reply nat-lab/docker-compose.yml]: Added comment

**Comment by stalowyjez** [comment]
lgtm, +1

**Review by stalowyjez — APPROVED** [review]

**Pull request merged**